pyglet/libs/win32/__init__.py

```diff
@@ -319,6 +280,37 @@
 _oleaut32.VariantClear.restype = HRESULT
 _oleaut32.VariantClear.argtypes = [c_void_p]
 
+if _debug_win32:
+    import traceback
+
+    _log_win32 = open('debug_win32.log', 'w')
```

Comment:
We could quiet this up by using atexit. Something like:
```py
import atexit
atexit.register(file.close)
```
I'm not sure how important it is though, since text files use line-buffering by default (I think).

Reply:
The file was never closed in the initial code. It's fine to put it at exit if you want.